docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
```

Comment:
Ports used are prefixed with a `1` so that if you want to run it locally, it will not clash with the normal firebase ports